robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -280,6 +280,16 @@ def add_batch_table_columns(self,*args):
         print (f'verify locator is {verify_field}')
         self.selenium.wait_until_page_does_not_contain_element(verify_field)
 
+    def return_template_builder_titles(self,page=None):
+        """Scrapes the specified page for the form field titles or section titles based on the argument passed and
+        stores them for recalling."""
```

Comment:
As you are looking for specific values to be passed for arguments please mention them specifically in the documentation. Also, recommend adding an else statement and raising an exception if any other values are passed to avoid false positives. For example like this
else:
    raise Exception("the title passed is invalid")

Reply:
Also, instead of scrapes, gets a list of field titles or section titles on the page would be more understandable.

Reply:
Fixed